test/e2e/kubernetes/node/node.go

```diff
@@ -65,7 +65,7 @@ type Address struct {
 type Info struct {
 	ContainerRuntimeVersion string `json:"containerRuntimeVersion"`
 	KubeProxyVersion        string `json:"kubeProxyVersion"`
-	KubeletProxyVersion     string `json:"kubeletVersion"`
```

Comment:
Not sure what happened here, it doesn't make sense for this property to be named `KubeletProxyVersion`, not to mention it deviates from the k8s type def (and it's not being used in the E2E code anywhere).